### ISSUE TITLE
Translation report should have statistics for shared strings

### ISSUE THREAD

**Issue by liammulh** [phetsims/rosetta#347]
For https://github.com/phetsims/qa/issues/869. There is a little-known third category of strings – shared strings. The other two categories are sim-specific and common. The vast majority of sims only have sim-specific and common strings, however, there are some sims that have all three categories. Shared strings are strings that are shared between sims, but don't belong to common repositories such as phetsims/scenery, phetsims/joist, etc. For example, Concentration has shared strings with Beer's Law Lab.

I think we should show shared strings in the translation "report", but I need to know the design. We could do something like:

| Sim-Specific Strings | Shared Strings | Common Strings |
| - | - | - |
| x% (y of z) | n/a | x% (y of z) |
| x% (y of z) | x% (y of z) | x% (y of z) |

However, this might be a bit strange since most sims would have "n/a" in their shared strings column. @oliver-phet, what are your thoughts on how we ought to design this?

**Comment by liammulh** [comment]
@jbphet and I are going to proceed with "N/A" for now, but we can change it.

The percentage associated with the "N/A" cells should either be 100% or 0%, depending on how we want the sort to work. (Presumably we want to be able to sort this column as well.)

**Comment by oliver-phet** [comment]
Here's what I think would work:
**On the "Sim List" screen:**
Combine sim-specific and shared strings into the existing "Sim-Specific Strings" and rename to column "Sim Strings"

**On the "Sim Translation" screen:**
Separate the sim-specific and shared strings like you proposed:
| Sim-Specific Strings | Shared Strings | Common Strings |
| - | - | - |
| x% (y of z) | N/A | x% (y of z) |
| x% (y of z) | x% (y of z) | x% (y of z) |

**Comment by liammulh** [comment]
As of https://github.com/phetsims/rosetta/commit/ff73dc67e550df80c9354a99caf526864bbb453b, I believe this is done. Marking as ready for review, although if we end up using the "composite" column, this won't really need to be reviewed. In any case, I think the work is done for this issue.

**Comment by Nancy-Salpepi** [comment]
In Round 2, the Shared Strings percentage only appears in one of the sims. For example:
<img width="681" alt="Screenshot 2023-01-20 at 8 43 37 AM" src="https://user-images.githubusercontent.com/87318828/213710399-e699534b-6c81-49e0-9ce7-3a99d3571381.png">

@liammulh shouldn't that stat appear in both sims?

**Comment by liammulh** [comment]
> Looking into this more, it seems like "shared strings" are only listed in the 'daughter' sim (for example, a basics version). Is that correct? If so, this issue can be closed.

Yep, that's how it works. Shared strings is kind of a misnomer. They are shared, but it's a one-way sharing.
